tests/test_dependencies.py

```diff
@@ -0,0 +1,57 @@
+import pytest
+
+import unblob.dependencies as dependencies
+
+
+class TestDependencies:
+
+    DEPENDENCIES = ["a", "bb", "cccccccccccccccc"]
+
+    @pytest.fixture(autouse=True)
+    def mock_dependencies(self, monkeypatch):
+        monkeypatch.setattr(dependencies, "get_dependencies", lambda: self.DEPENDENCIES)
+
+    def test_get_dependencies(self):
+        # This only tests that "mock_dependencies" was successful
+        assert dependencies.get_dependencies() == self.DEPENDENCIES
+
+    @pytest.mark.parametrize(
+        "command, expected",
+        (
+            ("some_random_command_surely_not_found", False),
+            ("which", True),
+        ),
+    )
+    def test_is_installed(self, command, expected):
+        assert expected == dependencies._is_installed(command)
+
+    @pytest.mark.parametrize("found", (True, False))
+    def test_get_dependency_map(self, monkeypatch, found):
+        monkeypatch.setattr(dependencies, "_is_installed", lambda command: found)
```

Comment:
We should talk about testing 1x1. Tests should be written for public API only. That makes refactorings very easy.
Sometimes we can break that if we have a very complex function, but this is not the case here.